models/comments.models.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../db/connection");
+
+const deleteCommentById = (comment_id) => {
+  return db
+    .query("DELETE FROM comments WHERE comment_id = $1 RETURNING *;", [
+      comment_id,
+    ])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "Comment Not Found",
+        });
+      }
+      return rows[0];
```

Comment:
Don't need to return rows[0] here as successful DELETE responses only send back a status code/not being used in controller